CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ include(GNUInstallDirs)
 message(STATUS "\n\n====== Finding 3rd Party Packages ======\n")
 
 find_package(ament_cmake REQUIRED)
-find_package(gflags REQUIRED)
```

Comment:
This needs to be moved to src/applications/CMakelists.txt

Reply:
Done.